runtime/help/options.md

```diff
@@ -40,65 +40,65 @@ Here are the available options:
    the backup directory. Backups are made in the background for newly modified
    buffers every 8 seconds, or when micro detects a crash.
 
-    default value: `true`
+   default value: `true`
 
 * `backupdir`: the directory micro should place backups in. For the default
    value of `""` (empty string), the backup directory will be
    `ConfigDir/backups`, which is `~/.config/micro/backups` by default. The
    directory specified for backups will be created if it does not exist.
 
-    default value: `""` (empty string)
+   default value: `""` (empty string)
 
 * `basename`: in the infobar and tabbar, show only the basename of the file
    being edited rather than the full path.
 
-    default value: `false`
+   default value: `false`
 
 * `clipboard`: specifies how micro should access the system clipboard.
    Possible values are:
-    * `external`: accesses clipboard via an external tool, such as xclip/xsel
```

Comment:
> If first word started from ` it will be 3 otherwise 2

Wasn't it "agreed" to be like this?
The `external` is on the indentation level of 3 due to the upper level started with `.

Reply:
I was talking here about "sub-bullet" lines (beginning with `*` *within* an option description), which are a special type of formatting, so the existing style for them (4 spaces indentation, instead of 3 spaces like for "normal text") makes sense to me, so I was merely saying that I see no reason to change it from 4 to 3 spaces for sub-bullets. (Although there are is not much "convention" in this case anyway, so I'm fine with changing them to 3 spaces if you insist. I don't like arguing about such trivia either.)